src/views/workflow-page/config/workflow-page-tabs.config.ts

```diff
@@ -16,6 +19,7 @@ const workflowPageTabsConfig = [
   {
     key: 'history',
     title: 'History',
+    endEnhancer: createElement(WorkflowPagePendingEventsBadge),
```

Comment:
better if can support passing the element with/without createElement. Usually we need createElement if we are sending props other wise we don't need it